adafruit_ble_apple_media.py

```diff
@@ -50,7 +52,7 @@ class _RemoteCommand(ComplexCharacteristic):
     uuid = VendorUUID("9B3C81D8-57B1-4A8A-B8DF-0E56F7CA51C2")
 
     def __init__(self):
-        super().__init__(properties=Characteristic.WRITE_NO_RESPONSE | Characteristic.NOTIFY,
+        super().__init__(properties=Characteristic.WRITE_NO_RESPONSE if Characteristic.WRITE_NO_RESPONSE else Characteristic.NOTIFY,
```

Comment:
So I'm extra not sure about this one. I wasn't entirely sure what that bitwise or was doing, but I thought it was trying to conditionally assign either WRITE_NO_RESPONSE or NOTIFY based on which one was True. Sphinx really didn't like that so I changed it

Reply:
The properties are a bitmask, so it's doing a bitwise OR to set both bits.
`Characteristic.WRITE_NO_RESPONSE if Characteristic.WRITE_NO_RESPONSE else Characteristic.NOTIFY` is `a if a else b`, which is not what is intended. The values are all non-zero.

Reply:
@dhalbert Ok. Why I was changing them was that when I tried to build the docs, it gave me this error on all of the three that did a bitwise or
`TypeError: unsupported operand type(s) for |: 'WRITE_NO_RESPONSE' and 'NOTIFY'`

Reply:
Thanks for your review by the way. That was actually very helpful as I wasn't really sure what that line was doing

Reply:
They are just integers, taken from `_bleio`, so I think this is due to not mocking `_bleio` properly or something.

https://github.com/adafruit/Adafruit_CircuitPython_BLE/blob/3d7bb01776aaa06cb6455bfcb437d09d456281d8/adafruit_ble/characteristics/__init__.py#L84-L89

Reply:
Ok. I'll try to see about fixing that

Reply:
Aha, found it, here is a mock for `_bleio` in `adafruit_ble`: https://github.com/adafruit/Adafruit_CircuitPython_BLE/blob/master/docs/mocks/_bleio.py. And that mock is made available by these lines in `conf.py`:
```
docs/conf.py:6:sys.path.insert(0, os.path.abspath('mocks'))
docs/conf.py:24:autodoc_mock_imports = ["board"]
```

Reply:
Sorry for the jargon. A mock is a fake version of something, that acts enough like it to work for testing or whatever.

Reply:
@dhalbert Ok. So I should just copy that file over to this repo and enable it, right?

Reply:
Sorry for not responding quicker. I'm bad at checking notifications when I'm really focused on something

Reply:
No problem, I'm not sitting around waiting :smiley:. Right, I think you just need `sys.path.insert`, and the file in `mocks`, but do you also need `adafruit-circuitpython-ble` in `requirements.txt`? I haven't spent a lot of time on kind of thing. Maybe that's just for PyPi, and we don't need it now because we don't support BLE in blinka, but it should be there for later, I think.

Pinging  @kattni and/or @siddacious for `requirements.txt` q ^^

Reply:
@dhalbert @dherrada yes, the `requirements.txt` wants `adafruit-circuitpython-ble`, I believe as Dan suggests to future proof for Blinka

Reply:
I added it since it is imported in the module